main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"net/http"
+
+	handler "github.com/livepeer/leaderboard-serverless/api"
+	"github.com/livepeer/leaderboard-serverless/common"
+)
+
+// this func is for running in local mode.  Vercel does not use this as an entrypoint
+// so any logic here should only reflect what is needed for local development
+func main() {
+
```

Comment:
Could you check here if the DB is available + execute the migrations? I think it's not good that here the program starts without any errors, but them everything happens during the first API call.

Reply:
This is how the system was originally designed and also is required for it to work with the current Vercel deployment.  If changes were to be made, we would need to support the approach you are suggesting and one that is compatible with Vercel (without purchasing additional services from them).  My preference would be to leave it as is in order to avoid having two flows, one in this main function for local mode and another for Vercel serverless deployment.